### ISSUE TITLE
device notifier icons too large

### ISSUE THREAD

**Issue by joaovsilva** [manjaro/artwork-breath#6]
I'm using Maia icons over Breath Look and Feel and Desktop Theme. The Device Notifier actions' icons appear extremely large. Might it be something not right with the theme? Please take a look at the attached pic. From left to right: just clicked the device notifier icon on the tray; next, clicked over the device name (showing 'Open on Gwenview'); next, scrolled down (showing 'Open on file manager')
![device_notifier_maia_icons_in_breath_theme](https://cloud.githubusercontent.com/assets/27526859/25688952/5ae58be0-307b-11e7-81e3-268e17b0c7c9.png)

**Comment by mcder3** [comment]
Have you restarted the plasmashell process to see if the problem still persists?

**Comment by joaovsilva** [comment]
I've misread my pacman log and was convinced Breath was updated very recently when in fact it wasn't. But kiconthemes was and it's since then I notice this behaviour. I then performed the tests I should've done in the first place: changed themes and icons several times and noticed in fact this effect comes and goes randomly with any theme or icon set. I'm reporting to KDE. Sorry to have wasted your time!
